code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -15,36 +15,6 @@
 	/*food_flags = FOOD_FINGER_FOOD*/
 	w_class = WEIGHT_CLASS_TINY
 
-/obj/item/reagent_containers/food/snacks/candy/bronx
-	name = "South Bronx Paradise bar"
-	desc = "Lose weight, guaranteed! Caramel Mocha Flavor. Something about product consumption..."
-	icon_state = "bronx"
-	item_state = "candy"
-	trash = /obj/item/trash/candy
-	list_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/sugar = 2, /datum/reagent/yuck = 1)
-	junkiness = 10
-	bitesize = 10
-	filling_color = "#e4d4b7"
-	tastes = list("candy" = 5, "weight loss" = 4, "insect larva" = 1)
-	foodtype = JUNKFOOD | RAW | GROSS
-	custom_premium_price = 800
-	var/revelation = FALSE
-
-/obj/item/reagent_containers/food/snacks/candy/bronx/On_Consume(mob/living/eater)
-	. = ..()
-	if(ishuman(eater))
-		var/mob/living/carbon/human/carl = eater
-		var/datum/disease/P = new /datum/disease/parasite()
```

Comment:
Vaporize the disease too please it sucks
"Your liver is coming out one way or another" is stupif